src/devtools/client/debugger/src/components/SecondaryPanes/NewScopes.tsx

```diff
@@ -0,0 +1,112 @@
+import { Suspense } from "react";
+import { useAppSelector } from "ui/setup/hooks";
+import { getPauseId, getSelectedFrameId } from "../../selectors";
+import { getScopesSuspense } from "ui/suspense/scopeCache";
+import { Redacted } from "ui/components/Redacted";
+import InspectorContextReduxAdapter from "devtools/client/debugger/src/components/shared/InspectorContextReduxAdapter";
+import ErrorBoundary from "bvaughn-architecture-demo/components/ErrorBoundary";
+import Loader from "bvaughn-architecture-demo/components/Loader";
+import Expandable from "bvaughn-architecture-demo/components/Expandable";
+import Inspector from "bvaughn-architecture-demo/components/inspector";
+import { PauseId } from "@replayio/protocol";
+import { getAsyncParentPauseIdIfCached } from "ui/suspense/util";
+
+import styles from "./NewObjectInspector.module.css";
+import { ConvertedScope, convertScopes } from "../../utils/pause/scopes/convertScopes";
+import { getFramesSuspense } from "ui/suspense/frameCache";
+
+function ScopesRenderer() {
+  //TODO title for Window scope
+  //TODO isCurrentTimeInLoadedRegion
+  //TODO isErrored
+  const pauseId = useAppSelector(getPauseId);
+  const frameId = useAppSelector(getSelectedFrameId);
+  if (!pauseId || !frameId) {
+    return null; //TODO?
+  }
+
+  const splitFrameId = frameId.split(":");
```

Comment:
:speech_balloon: purely stylistically, I'd lean towards `const [groupIndex, frameIndex] = frameId.split(":")` (would still need to convert to a number)